src/utils/index.ts

```diff
@@ -113,7 +113,7 @@ export function currentFile(document?: vscode.TextDocument): CurrentFile {
     name = uri.path;
   } else if (fileExt === "cls") {
     // Allow Unicode letters
-    const match = content.match(/^[ \t]*Class (%?[\p{L}\d]+(?:\.[\p{L}\d]+)+)/imu);
+    const match = content.match(/^[ \t]*Class +(%?[\p{L}\d]+(?:\.[\p{L}\d]+)+)/imu);
```

Comment:
Maybe allow for Tabs as well as spaces?

```suggestion
    const match = content.match(/^[ \t]*Class[ \t]+(%?[\p{L}\d]+(?:\.[\p{L}\d]+)+)/imu);
```